traffic_ops/traffic_ops_golang/deliveryservice/servers/servers.go

```diff
@@ -1024,14 +987,16 @@ SELECT
   ds.cacheurl,
   ds.max_origin_connections,
   ds.topology,
-  ds.cdn_id
+  ds.cdn_id,
+  ds.multi_site_origin
 FROM
   deliveryservice ds
   JOIN type tp ON ds.type = tp.id
 `
 
 func scanDSInfoRow(row *sql.Row) (DSInfo, bool, error) {
 	di := DSInfo{}
+	var useMSO *bool
```

Comment:
Since `deliveryservice.multi_site_origin` is nullable, this actually needs to be a bool pointer. Otherwise, it wouldn't be able to scan `nil` into `useMSO`